doc/source/toolkit/ui.rst

```diff
@@ -1,101 +1,97 @@
    connecting to an existing AEDT session.
 
    .. note::
-      If the Antenna Toolkit Wizard is launched from AEDT, the **AEDT Settings** tab does not appear
+      If the Antenna Toolkit Wizard is launched from AEDT, the **Settings** tab does not appear
       because the toolkit is directly connected to the specific AEDT session.
 
    .. image:: ../_static/settings.png
       :width: 800
       :alt: Settings tab
 
-   The wizard has a progress bar and a logger box, where you can see the status of every operation.
-   A red progress bar means that the toolkit is busy. Every operation must wait until the toolkit is released.
+   The wizard has a progress circle and a logger box, where you can see the status of every operation.
 
-   Once the wizard is connected to AEDT, you want to use the wizard's **Antennas** menu to load an antenna
-   template into the wizard's **Design** tab.
+   .. image:: ../_static/progress.png
+      :width: 800
+      :alt: Progress tab
+
+   You can choose different antennas from the **Antenna catalog** menu to load the antennas
+   template.
 
-   .. image:: ../_static/design_connected.png
+   .. image:: ../_static/antenna_catalog.png
       :width: 800
-      :alt: Antenna Toolkit Wizard, Design tab
+      :alt: Antenna catalog
 
-   For example, if you select **Antennas > Horn > Conical Corrugated**, the **Design** tab displays
-   the **Synthesis** button and antenna template:
+   .. image:: ../_static/antenna_catalog_2.png
+      :width: 800
+      :alt: Antenna catalog 2
 
-   .. image:: ../_static/corrugated_1.png
+   For instance, if you select **Antennas > Bowtie > Bowtie Normal**,
+   the central page is updated to the **Synthesis** page and it shows the antenna template:
+
+   .. image:: ../_static/antenna_template.png
       :width: 800
-      :alt: Antenna Toolkit Wizard, corrugated synthesis
+      :alt: Antenna template
 
-   In addition to the **Synthesis** button, the **Design** tab displays a **Create Hfss Model** button. This
-   button is unavailable if the Antenna Toolkit Wizard is not connected to AEDT.
+   The user has two options: **Synthesis** and **Generate**.
+   **Generate** button is unavailable if the wizard is not connected to AEDT.
 
    - The **Synthesis** button is for performing the synthesis of the antenna. It does not need AEDT.
      You can see the parameters that control the antenna geometry. Additionally, you can do as many
      syntheses as you want and even change the antenna template.
 
-     .. image:: ../_static/corrugated_synthesis.png
+     .. image:: ../_static/antenna_synthesis.png
         :width: 800
-        :alt: Antenna Toolkit Wizard, corrugated synthesis
+        :alt: Antenna synthesis
 
-   - The **Create HFSS Model** button is for creating an HFSS model. It uses the **3D Component**,
+   - The **Generate** button is for creating a HFSS model. It uses the **3D Component**,
```

Comment:
```suggestion
   - The **Generate** button is for creating an HFSS model. It uses the **3D Component**,
```
You use "an" if the following letter is pronounced like a vowel-like the "eh" sound here.